content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,131 @@
+
+{{< image width="100%"
+    link="total-cpu.png"
+    caption="Total CPU usage for sidecar and ambient scenarios"
+    >}}
+
+{{< image width="100%"
+    link="total-memory.png"
+    caption="Total memory usage for sidecar and ambient scenarios"
+    >}}
+
+{{< image width="100%"
+    link="cpu-by-pod-stacked.png"
+    caption="Stacked CPU usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="memory-by-pod-stacked.png"
+    caption="Stacked memory usage by pod for sidecar and ambient pods"
+    >}}
+
+Looking at total CPU and memory utilization, we have to remember that in sidecar scenarios there are 144 sidecar containers required (48 services, 3 replicas each), while in ambient, only three ztunnel containers and 12 waypoint proxies are required. The stacked CPU and memory by pod graphs are excellent at highlighting just how many additional containers are present between scenarios. Memory usage of the Istio dataplane in the ztunnel-only ambient scenario sees *more than 99% savings* compared to sidecar scenarios, and 89% when waypoints are added. Looking at CPU, ztunnel sees *82% savings* compared to sidecar scenarios, and 40% when waypoints are deployed.
+
+Finally, let’s consider allocation since the graphs above have only covered usage. Every sidecar resource has a default request of 100 millicores vCPU and 128 MB memory. For simplicity, we assume ztunnels and waypoint proxies have similar requests and limits as their sidecar counterparts. Breaking it down, that’s a 98% reduction in allocated resources with ztunnel, and 90% when waypoints are included.
+
+{{< image width="100%"
+    link="allocation.png"
+    caption="CPU and memory allocation and cost breakdown for sidecar and ambient scenarios"
+    >}}
+
+Going further, we can calculate a dollar amount for these numbers by referring to [GCP monthly pricing](https://cloud.google.com/compute/vm-instance-pricing) for a monthly cost per GB memory and per CPU. Consider these two different machines and their costs:
+
+- `n2-standard-4` (4 CPU, 16 GB) at $141/month
+- `n2-highmem-4` (4 CPU, 32 GB) at $191/month
+
+Calculating the difference in memory results in 1GB costing $3.33/month. Similarly for CPU:
+
+- `n2-standard-4` (4 CPU, 16 GB) at $141/month
+- `n2-highmem-2` (2 CPU, 16 GB) at $95/month
+
+Calculating the difference results in 1 CPU costing approximately $23/month. We can literally put a dollar amount on the savings ambient brings!
```

Comment:
@GregHanson chatting with @howardjohn offline, this sentence made him to think saving is $23 per month.   Maybe no need to highlight this?  We just need to say A CPU cost X, we use Y CPUs, so we cost Z a month (for sidecars or ambient L4 or L4 & L7).

Reply:
updated, PTAL